test/language/expressions/async-generators/early-errors-next-receiver-not-async-iterator.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Benjamin Gruenbaum <benjamingr@gmail.com>
+esid: pending
+description: >
+  next() rejects with a TypeError when receiver is not an AsyncGenerator instance:
+  phase: early
```

Comment:
The phase and type frontmatter bits should be removed, since this is not really a negative test